### PR TITLE
Metadata editor tests

### PR DESCRIPTION
**Pull request by solth**
This pull request adds tests for the metadata editor.

Part of https://github.com/orgs/kitodo/projects/8

### CODE REVIEW

**Comment by solth** [comment]
> Just a question: is it possible to change the Junit4 to Junit5 tests in the new test classes? Changing it could be fast done but sometimes even tricky and maybe with a lot of time.

Good point. I will try to adjust the tests accordingly and re-open the PR for review afterwards.

**Comment by solth** [comment]
@henning-gerhardt I updated the tests to JUnit5.

File: Kitodo/src/test/java/org/kitodo/production/forms/dataeditor/DataEditorFormTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.kitodo.api.MetadataEntry;
+import org.kitodo.api.dataformat.LogicalDivision;
+import org.kitodo.api.dataformat.PhysicalDivision;
+import org.kitodo.api.dataformat.View;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/production/forms/dataeditor/DataEditorFormTest.java:25]: Wrong order of import statements.

File: Kitodo/src/test/java/org/kitodo/production/services/data/DataEditorServiceIT.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+import org.kitodo.data.database.beans.Ruleset;
+import org.kitodo.data.database.exceptions.DAOException;
+import org.kitodo.data.exceptions.DataException;
+import org.kitodo.production.forms.createprocess.ProcessFieldedMetadata;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.dataeditor.DataEditorService;
+import org.kitodo.test.utils.ProcessTestUtils;
+import org.primefaces.model.DefaultTreeNode;
+
+import javax.faces.model.SelectItem;
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/production/services/data/DataEditorServiceIT.java:43]: Wrong order of import statements.

File: Kitodo/src/test/java/org/kitodo/production/forms/dataeditor/DataEditorFormTest.java [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    private final static String TITLE_METADATA_KEY = "maintitle";
+
+    /**
+     * Test consecutive page selection in data editor form.
+     */
+    @Test
+    public void shouldCheckIfConsecutivePagesAreSelected() {
+        DataEditorForm dataEditorForm = new DataEditorForm();
+        dataEditorForm.setSelectedMedia(prepareSelectedMedia(true, false));
+        Assertions.assertTrue(dataEditorForm.consecutivePagesSelected(),
```
Comment (henning-gerhardt) [inline Kitodo/src/test/java/org/kitodo/production/forms/dataeditor/DataEditorFormTest.java:39]: Maybe import the Assertions.* as static imports?

**Review by henning-gerhardt — CHANGES_REQUESTED** [review]
The order of import statements should be correct and maybe import the different Assertions as static import as this should improve the reading quality of code.

**Pull request merged**